### ISSUE TITLE
2025-04-01, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#127]
# Today's Arxiv Papers

(1) [Any2Caption:Interpreting Any Condition to Caption for Controllable Video Generation](http://arxiv.org/abs/2503.24379v1), 2025-03-31

(2) [Harnessing the Reasoning Economy: A Survey of Efficient Reasoning for Large Language Models](http://arxiv.org/abs/2503.24377v1), 2025-03-31

(3) [Exploring the Effect of Reinforcement Learning on Video Understanding: Insights from SEED-Bench-R1](http://arxiv.org/abs/2503.24376v1), 2025-03-31

(4) [Effectively Controlling Reasoning Models through Thinking Intervention](http://arxiv.org/abs/2503.24370v1), 2025-03-31

(5) [SQuat: Subspace-orthogonal KV Cache Quantization](http://arxiv.org/abs/2503.24358v1), 2025-03-31

(6) [ORAL: Prompting Your Large-Scale LoRAs via Conditional Recurrent Diffusion](http://arxiv.org/abs/2503.24354v1), 2025-03-31

(7) [BEATS: Bias Evaluation and Assessment Test Suite for Large Language Models](http://arxiv.org/abs/2503.24310v1), 2025-03-31

(8) [A Systematic Evaluation of LLM Strategies for Mental Health Text Analysis: Fine-tuning vs. Prompt Engineering vs. RAG](http://arxiv.org/abs/2503.24307v1), 2025-03-31

(9) [Is analogy enough to draw novel adjective-noun inferences?](http://arxiv.org/abs/2503.24293v1), 2025-03-31

(10) [Rec-R1: Bridging Generative Large Language Models and User-Centric Recommendation Systems via Reinforcement Learning](http://arxiv.org/abs/2503.24289v1), 2025-03-31

(11) [Evaluating and Designing Sparse Autoencoders by Approximating Quasi-Orthogonality](http://arxiv.org/abs/2503.24277v1), 2025-03-31

(12) [Enhancing Large Language Models (LLMs) for Telecommunications using Knowledge Graphs and Retrieval-Augmented Generation](http://arxiv.org/abs/2503.24245v1), 2025-03-31

(13) [What, How, Where, and How Well? A Survey on Test-Time Scaling in Large Language Models](http://arxiv.org/abs/2503.24235v1), 2025-03-31

(14) [PAARS: Persona Aligned Agentic Retail Shoppers](http://arxiv.org/abs/2503.24228v1), 2025-03-31

(15) [Synthetic News Generation for Fake News Classification](http://arxiv.org/abs/2503.24206v1), 2025-03-31

(16) [TwT: Thinking without Tokens by Habitual Reasoning Distillation with Multi-Teachers' Guidance](http://arxiv.org/abs/2503.24198v1), 2025-03-31

(17) [Text2Tracks: Prompt-based Music Recommendation via Generative Retrieval](http://arxiv.org/abs/2503.24193v1), 2025-03-31

(18) [Output Constraints as Attack Surface: Exploiting Structured Generation to Bypass LLM Safety Mechanisms](http://arxiv.org/abs/2503.24191v1), 2025-03-31

(19) [Implicit In-Context Learning: Evidence from Artificial Language Experiments](http://arxiv.org/abs/2503.24190v1), 2025-03-31

(20) [LLM4FS: Leveraging Large Language Models for Feature Selection and How to Improve It](http://arxiv.org/abs/2503.24157v1), 2025-03-31

(21) [TeleAntiFraud-28k: A Audio-Text Slow-Thinking Dataset for Telecom Fraud Detection](http://arxiv.org/abs/2503.24115v1), 2025-03-31

(22) [Grounding Agent Reasoning in Image Schemas: A Neurosymbolic Approach to Embodied Cognition](http://arxiv.org/abs/2503.24110v1), 2025-03-31

(23) [Is LLM the Silver Bullet to Low-Resource Languages Machine Translation?](http://arxiv.org/abs/2503.24102v1), 2025-03-31

(24) [DANTE-AD: Dual-Vision Attention Network for Long-Term Audio Description](http://arxiv.org/abs/2503.24096v1), 2025-03-31

(25) [TransMamba: Flexibly Switching between Transformer and Mamba](http://arxiv.org/abs/2503.24067v1), 2025-03-31

(26) [Artificial Conversations, Real Results: Fostering Language Detection with Synthetic Data](http://arxiv.org/abs/2503.24062v1), 2025-03-31

(27) [ReaLM: Reliable and Efficient Large Language Model Inference with Statistical Algorithm-Based Fault Tolerance](http://arxiv.org/abs/2503.24053v1), 2025-03-31

(28) [Towards Scientific Intelligence: A Survey of LLM-based Scientific Agents](http://arxiv.org/abs/2503.24047v1), 2025-03-31

(29) [Pay More Attention to the Robustness of Prompt for Instruction Data Mining](http://arxiv.org/abs/2503.24028v1), 2025-03-31

(30) [IntelliCircos: A Data-driven and AI-powered Authoring Tool for Circos Plots](http://arxiv.org/abs/2503.24021v1), 2025-03-31

(31) [H2VU-Benchmark: A Comprehensive Benchmark for Hierarchical Holistic Video Understanding](http://arxiv.org/abs/2503.24008v1), 2025-03-31

(32) [Rethinking Key-Value Cache Compression Techniques for Large Language Model Serving](http://arxiv.org/abs/2503.24000v1), 2025-03-31

(33) [Rubric Is All You Need: Enhancing LLM-based Code Evaluation With Question-Specific Rubrics](http://arxiv.org/abs/2503.23989v1), 2025-03-31

(34) [Green MLOps to Green GenOps: An Empirical Study of Energy Consumption in Discriminative and Generative AI Operations](http://dx.doi.org/10.3390/info16040281), 2025-03-31

(35) [Model Hemorrhage and the Robustness Limits of Large Language Models](http://arxiv.org/abs/2503.23924v1), 2025-03-31

(36) [Entropy-Based Adaptive Weighting for Self-Training](http://arxiv.org/abs/2503.23913v1), 2025-03-31

(37) [Rubrik's Cube: Testing a New Rubric for Evaluating Explanations on the CUBE dataset](http://arxiv.org/abs/2503.23899v1), 2025-03-31

(38) [Better wit than wealth: Dynamic Parametric Retrieval Augmented Generation for Test-time Knowledge Enhancement](http://arxiv.org/abs/2503.23895v1), 2025-03-31

(39) [SchemaAgent: A Multi-Agents Framework for Generating Relational Database Schema](http://arxiv.org/abs/2503.23886v1), 2025-03-31

(40) [GenSwarm: Scalable Multi-Robot Code-Policy Generation and Deployment via Language Models](http://arxiv.org/abs/2503.23875v1), 2025-03-31

(41) [Exploring In-Context Learning Capabilities of ChatGPT for Pathological Speech Detection](http://arxiv.org/abs/2503.23873v1), 2025-03-31

(42) [Communication-Efficient and Personalized Federated Foundation Model Fine-Tuning via Tri-Matrix Adaptation](http://arxiv.org/abs/2503.23869v1), 2025-03-31

(43) [Evaluating small vision-language models as AI assistants for radio astronomical source analysis tasks](http://arxiv.org/abs/2503.23859v1), 2025-03-31

(44) [SpeechDialogueFactory: Generating High-Quality Speech Dialogue Data to Accelerate Your Speech-LLM Development](http://arxiv.org/abs/2503.23848v1), 2025-03-31

(45) [OrchMLLM: Orchestrate Multimodal Data with Batch Post-Balancing to Accelerate Multimodal Large Language Model Training](http://arxiv.org/abs/2503.23830v1), 2025-03-31

(46) [Expanding RL with Verifiable Rewards Across Diverse Domains](http://arxiv.org/abs/2503.23829v1), 2025-03-31

(47) [Aud-Sur: An Audio Analyzer Assistant for Audio Surveillance Applications](http://arxiv.org/abs/2503.23827v1), 2025-03-31

(48) [MVDRAM: Enabling GeMV Execution in Unmodified DRAM for Low-Bit LLM Acceleration](http://arxiv.org/abs/2503.23817v1), 2025-03-31

(49) [Did ChatGPT or Copilot use alter the style of internet news headlines? A time series regression analysis](http://arxiv.org/abs/2503.23811v1), 2025-03-31

(50) [Get the Agents Drunk: Memory Perturbations in Autonomous Agent-based Recommender Systems](http://arxiv.org/abs/2503.23804v1), 2025-03-31

(51) [Thinking Longer, Not Larger: Enhancing Software Engineering Agents via Scaling Test-Time Compute](http://arxiv.org/abs/2503.23803v1), 2025-03-31

(52) [Adaptive Layer-skipping in Pre-trained LLMs](http://arxiv.org/abs/2503.23798v1), 2025-03-31

(53) [LLMigrate: Transforming "Lazy" Large Language Models into Efficient Source Code Migrators](http://arxiv.org/abs/2503.23791v1), 2025-03-31

(54) [ObfusQate: Unveiling the First Quantum Program Obfuscation Framework](http://arxiv.org/abs/2503.23785v1), 2025-03-31

(55) [DebFlow: Automating Agent Creation via Agent Debate](http://arxiv.org/abs/2503.23781v1), 2025-03-31

(56) [WinoWhat: A Parallel Corpus of Paraphrased WinoGrande Sentences with Common Sense Categorization](http://arxiv.org/abs/2503.23779v1), 2025-03-31

(57) [CONGRAD:Conflicting Gradient Filtering for Multilingual Preference Alignment](http://arxiv.org/abs/2503.23777v1), 2025-03-31

(58) [XLRS-Bench: Could Your Multimodal LLMs Understand Extremely Large Ultra-High-Resolution Remote Sensing Imagery?](http://arxiv.org/abs/2503.23771v1), 2025-03-31

(59) [STI-Bench: Are MLLMs Ready for Precise Spatial-Temporal World Understanding?](http://arxiv.org/abs/2503.23765v1), 2025-03-31

(60) [UniSep: Universal Target Audio Separation with Language Models at Scale](http://arxiv.org/abs/2503.23762v1), 2025-03-31

(61) [Short-video Propagation Influence Rating: A New Real-world Dataset and A New Large Graph Model](http://arxiv.org/abs/2503.23746v1), 2025-03-31

(62) [LANID: LLM-assisted New Intent Discovery](http://arxiv.org/abs/2503.23740v1), 2025-03-31

(63) [AdaMMS: Model Merging for Heterogeneous Multimodal Large Language Models with Unsupervised Coefficient Optimization](http://arxiv.org/abs/2503.23733v1), 2025-03-31

(64) [Detecting Functional Bugs in Smart Contracts through LLM-Powered and Bug-Oriented Composite Analysis](http://arxiv.org/abs/2503.23718v1), 2025-03-31

(65) [HOIGen-1M: A Large-scale Dataset for Human-Object Interaction Video Generation](http://arxiv.org/abs/2503.23715v1), 2025-03-31

(66) [Building Instruction-Tuning Datasets from Human-Written Instructions with Open-Weight Large Language Models](http://arxiv.org/abs/2503.23714v1), 2025-03-31

(67) [A Conceptual Framework for Human-AI Collaborative Genome Annotation](http://arxiv.org/abs/2503.23691v1), 2025-03-31

(68) [Mapping Geopolitical Bias in 11 Large Language Models: A Bilingual, Dual-Framing Analysis of U.S.-China Tensions](http://arxiv.org/abs/2503.23688v1), 2025-03-31

(69) [MKA: Leveraging Cross-Lingual Consensus for Model Abstention](http://arxiv.org/abs/2503.23687v1), 2025-03-31

(70) [Large Language Models Pass the Turing Test](http://arxiv.org/abs/2503.23674v1), 2025-03-31

(71) [WHERE and WHICH: Iterative Debate for Biomedical Synthetic Data Augmentation](http://arxiv.org/abs/2503.23673v1), 2025-03-31

(72) [Context-Independent OCR with Multimodal LLMs: Effects of Image Resolution and Visual Complexity](http://arxiv.org/abs/2503.23667v1), 2025-03-31

(73) [DeepDubber-V1: Towards High Quality and Dialogue, Narration, Monologue Adaptive Movie Dubbing Via Multi-Modal Chain-of-Thoughts Reasoning Guidance](http://arxiv.org/abs/2503.23660v1), 2025-03-31

(74) [GIScience in the Era of Artificial Intelligence: A Research Agenda Towards Autonomous GIS](http://arxiv.org/abs/2503.23633v1), 2025-03-31

(75) [Exploring GPT-4 for Robotic Agent Strategy with Real-Time State Feedback and a Reactive Behaviour Framework](http://arxiv.org/abs/2503.23601v1), 2025-03-30

(76) [When LLM Therapists Become Salespeople: Evaluating Large Language Models for Ethical Motivational Interviewing](http://arxiv.org/abs/2503.23566v1), 2025-03-30

(77) [Whisper-LM: Improving ASR Models with Language Models for Low-Resource Languages](http://arxiv.org/abs/2503.23542v1), 2025-03-30

(78) [Question-Aware Knowledge Graph Prompting for Enhancing Large Language Models](http://arxiv.org/abs/2503.23523v1), 2025-03-30

(79) [If an LLM Were a Character, Would It Know Its Own Story? Evaluating Lifelong Learning in LLMs](http://arxiv.org/abs/2503.23514v1), 2025-03-30

(80) [RARE: Retrieval-Augmented Reasoning Modeling](http://arxiv.org/abs/2503.23513v1), 2025-03-30

(81) [SCORE: Story Coherence and Retrieval Enhancement for AI Narratives](http://arxiv.org/abs/2503.23512v1), 2025-03-30

(82) [Re-Aligning Language to Visual Objects with an Agentic Workflow](http://arxiv.org/abs/2503.23508v1), 2025-03-30

(83) [Evolutionary Prompt Optimization Discovers Emergent Multimodal Reasoning Strategies in Vision-Language Models](http://arxiv.org/abs/2503.23503v1), 2025-03-30

(84) [DNA and Human Language: Epigenetic Memory and Redundancy in Linear Sequence](http://arxiv.org/abs/2503.23494v1), 2025-03-30

(85) [POINT$^{2}$: A Polymer Informatics Training and Testing Database](http://arxiv.org/abs/2503.23491v1), 2025-03-30

(86) [Benchmarking Systematic Relational Reasoning with Large Language and Reasoning Models](http://arxiv.org/abs/2503.23487v1), 2025-03-30

(87) [Order Independence With Finetuning](http://arxiv.org/abs/2503.23483v1), 2025-03-30

(88) [Codehacks: A Dataset of Adversarial Tests for Competitive Programming Problems Obtained from Codeforces](http://arxiv.org/abs/2503.23466v1), 2025-03-30

(89) [VideoGen-Eval: Agent-based System for Video Generation Evaluation](http://arxiv.org/abs/2503.23452v1), 2025-03-30

(90) [Semantic-Preserving Transformations as Mutation Operators: A Study on Their Effectiveness in Defect Detection](http://arxiv.org/abs/2503.23448v1), 2025-03-30

(91) [Speculative End-Turn Detector for Efficient Speech Chatbot Assistant](http://arxiv.org/abs/2503.23439v1), 2025-03-30

(92) [CoRanking: Collaborative Ranking with Small and Large Ranking Agents](http://arxiv.org/abs/2503.23427v1), 2025-03-30

(93) [A Multi-agent Onboarding Assistant based on Large Language Models, Retrieval Augmented Generation, and Chain-of-Thought](http://arxiv.org/abs/2503.23421v1), 2025-03-30

(94) [An Analysis of Decoding Methods for LLM-based Agents for Faithful Multi-Hop Question Answering](http://arxiv.org/abs/2503.23415v1), 2025-03-30

(95) [Scaling Auditory Cognition via Test-Time Compute in Audio Language Models](http://arxiv.org/abs/2503.23395v1), 2025-03-30

(96) [ToRL: Scaling Tool-Integrated RL](http://arxiv.org/abs/2503.23383v1), 2025-03-30

(97) [KernelDNA: Dynamic Kernel Sharing via Decoupled Naive Adapters](http://arxiv.org/abs/2503.23379v1), 2025-03-30

(98) [FeRG-LLM : Feature Engineering by Reason Generation Large Language Models](http://arxiv.org/abs/2503.23371v1), 2025-03-30

(99) [Large Language Models Are Better Logical Fallacy Reasoners with Counterargument, Explanation, and Goal-Aware Prompt Formulation](http://arxiv.org/abs/2503.23363v1), 2025-03-30

(100) [Mixture of Routers](http://arxiv.org/abs/2503.23362v1), 2025-03-30
